tests/conftest.py

```diff
@@ -225,6 +196,67 @@ def set_basic_auth_logins(self, **jira_class_kwargs):
             self.jira_sysadmin = JIRA(self.CI_JIRA_URL, **jira_class_kwargs)
             self.jira_normal = JIRA(self.CI_JIRA_URL, **jira_class_kwargs)
 
+    def _project_exists(self, project_key: str) -> bool:
```

Comment:
seems like functionality to add to the library :-)

Reply:
should i add a feature request? :-D

Reply:
go for it !

Reply:
what i would do though is catch a more specific exception here. Which counts for all down the line actually.
to me it feels like JIRAError
this 'No project could be found' seems to be coming from jira itself ... 
similarily i would add a raise in an `else` statement if that string was not found

Reply:
if now you have a connection error it would return True

Reply:
now updated, once instance seems like it should be a genuine case, so I've left it.